src/helpers/generator.ts

```diff
@@ -22,9 +22,10 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import { Action, ActionContext } from "../common/types";
+import type { ActionInContext } from "@elastic/synthetics";
+import { Action } from "../common/types";
```

Comment:
a minor comment(probably it's just my preference so feel free to ignore).
 I wonder if we can export these two from the same source. `Action` type is derived from `ActionInContext` and seeing them imported from different sources seems like they are unrelated.

Reply:
Yes we can modify synthetics to also export the `Action` type. If we do this we'll want to hold off on merging this until the next release of the synthetics library, which is fine. This change is low-priority.